### DISCUSSION TITLE
Audi TT 2019 VC enable sat nav

### DISCUSSION THREAD

**Discussion by AdamiPL** [Mr-MIBonk/M.I.B._More-Incredible-Bash#203] [Q&A]
Hi

is it possible to enable sat nav with M.i.b at Audi TT 2019 with MHI2_ER_AU43x_P5098_MU1339..
right now theres no navigation icon.

**Comment by harman-f** [comment]
yes, that is possible.
Just get latest M.I.B and runn AIO patch.

**Reply by AdamiPL** [reply]
and that's it? because i already run the patch (script 3.5.1), audi smartphone interface appear but no navigation.

**Reply by clown522** [reply]
Any Update? Same problem after using AIO patch. Only CarPlay has appeared.
